### ISSUE TITLE
Dice Configuration from foundryvtt (v12) not reading"out of system"  rolls.

### ISSUE THREAD

**Issue by Top64** [fvtt-fria-ligan/blade-runner-foundry-vtt#54]
### Describe the bug

[Core] Dice configuration for system rolls sets for Bluetooth Dice.

[Pixels itself is a Module for conection[ The module work as intended in foundry (Rolling a Die send the message to the chat)

### Reproduction

Select [Pixels - Electronic Dice] as dice handler.

Character Sheet. Choose any rolling interaction. (Expect to see a promt) - system rolls directly.

### System Info

```sh
Blade Runner 12.0.1, Foundry v12 [331]
```

### Priority this issue should have?

None

### Validations

- [X] Done a clean install of the system.
- [X] Disabled all modules.
- [X] Checked if the issue is present in a new world instance (create a new world in foundry using the Blade Runner system).

**Comment by Stefouch** [comment]
Hi @Top64 ! Thank you for the bug report.

Do you have any errors thrown in the console log? You can see the console by typing <kbd>F12</kbd> in your internet browser.

**Comment by Top64** [comment]
@Stefouch 
![image](https://github.com/user-attachments/assets/68b1397a-2bee-442d-9d71-04a10887f511)

**Comment by Stefouch** [comment]
Anything in red color ?

**Comment by Top64** [comment]
Nope, only these warnings appears in the console. I will check others Year Zero system to take a look at this and give you more information.

**Comment by Top64** [comment]
Dice Handler working on Tales from the Loop and Alien RPG
![image](https://github.com/user-attachments/assets/a0b1bae5-5847-43c4-b754-623c6abe5456)

In the Blade Runner System, instead of showing the [waiting for rolling/inputs] it direcly rolls the dice.
![image](https://github.com/user-attachments/assets/42fd1b88-fbc4-4c31-833d-ba9fd87597e1)
